tinygrad/codegen/cstyle.py

```diff
@@ -120,8 +120,7 @@ def kk(s): kernel.append("  "*depth+s)
       # TODO: merge with CONST?
       if bufs[args.i] is not None and isinstance(bufs[args.i].realized, RawConst):
         assert newvar.ltype == LocalTypes.float, "const can't be float4"
-        # nan? inf?
-        val = f"{bufs[args.i].realized._buf}" + ("f" if not dtypes.is_int(bufs[args.i].dtype) else "")
+        val = str({float("inf"):"INFINITY", float("nan"):"NAN"}.get(bufs[args.i].realized._buf, bufs[args.i].realized._buf)) + ("f" if not dtypes.is_int(bufs[args.i].dtype) and not bufs[args.i].realized._buf in [float("inf"), float("nan")] else "")
```

Comment:
can we stop with golfed 250 char one-liners? It's going to be more readable if you use proper `if` and I'm sure it will be more fast without all of these intermediate dicts and lists